crisischeckin/crisicheckinweb/Web.config

```diff
@@ -31,8 +31,11 @@
     <add key="elmah.mvc.allowedUsers" value="*" />
     <add key="elmah.mvc.route" value="elmah" />
     <add key="environment" value="development" />
-    <add key="smtp.fromname" value="CrisisCheckin Dev"/>
-    <add key="smtp.fromaddress" value="no-reply@crisischeckin.com"/>
+    <add key="smtp.fromname" value="CrisisCheckin Dev" />
+    <add key="smtp.fromaddress" value="no-reply@crisischeckin.com" />
+    <add key="SMS.fromphone" value="+15005550006" /> <!-- This is Twilio Magic Number -->
+    <add key="twilio.account.sid" value="AC9513bcd4bad9abfe249caa4f9b96643d" />
```

Comment:
Here is where we are going to make the changes to hide the magic numbers, and make it so that in development, the code writes a text file. I'll add comments at the other locations as well.